frontend/src/component/common/Table/cells/LinkCell/LinkCell.tsx

```diff
@@ -26,23 +30,25 @@ export const LinkCell: React.FC<ILinkCellProps> = ({
     subtitle,
     children,
 }) => {
+    const subTitleClean = removeMd(subtitle);
```

Comment:
I think this casing is more consistent, since it matches the original name.

Alternatively, if it's more readable, we could use something like `plainSubtitle` or `plainTextSubtitle`.

```suggestion
    const subtitleClean = removeMd(subtitle);
```